src/openrct2/interface/InteractiveConsole.cpp

```diff
@@ -1687,6 +1701,7 @@ static constexpr const console_command console_command_table[] = {
                                     "Loading a scenery group will not load its associated objects.\n"
                                     "This is a safer method opposed to \"open object_selection\".",
                                     "load_object <objectfilenodat>" },
+    { "load_park", cc_load_park, "Load park", "load_park [name]" },
```

Comment:
Please mention it will read a file from user's `save` directory.

Reply:
Why is the name marked as optional? Makes no sense to me without the park name and it will fail anyway.

Reply:
This was a copy from save_park. I did not change that brackets. Sorry. should be just "nameofsave".